internal/pkg/template/repository/manifest/manifest.go

```diff
@@ -6,36 +6,43 @@ import (
 	"github.com/keboola/keboola-as-code/internal/pkg/build"
 	"github.com/keboola/keboola-as-code/internal/pkg/filesystem"
 	"github.com/keboola/keboola-as-code/internal/pkg/json"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
 	"github.com/keboola/keboola-as-code/internal/pkg/utils"
 	"github.com/keboola/keboola-as-code/internal/pkg/validator"
 )
 
 const FileName = `repository.json`
 
 type Manifest struct {
-	fs       filesystem.Fs
-	changed  bool
-	*Content `validate:"required,dive"` // content of the file, updated only on load/save
```

Comment:
`Content -> file`. 
Povodne tu bol obsah suboru cely cas pritomny, a menil sa iba pri load/save.
Spravil som to teraz lepsie, obsah suboru je v strukture `file`, a tu sa nakopiruje iba to co potrebujeme - > `templates`.